feeadjuster/test_feeadjuster.py

```diff
@@ -16,15 +14,19 @@ def test_feeadjuster_starts(node_factory):
     l1 = node_factory.get_node()
     # Test dynamically
     l1.rpc.plugin_start(plugin_path)
-    assert l1.daemon.is_in_log("Plugin feeadjuster initialized.*")
+    assert l1.daemon.wait_for_log("Plugin feeadjuster initialized.*")
```

Comment:
This is the change i don't get the rationale for. At this point it must already be in log. (and assert in unnecessary)

Reply:
Removed the assert.

Reply:
- ack on the assert ;)
- using `is_in_log` was flaky because it checks just right now at the time of python execution. If the plugin was not `init`ed yet then `is_in_log` failed whereas `wait_for_log` will wait until this happened.

Reply:
But `plugin start` waits for init iirc

Reply:
Nevermind, it's as good as it is anyways :)